pkg/codec.go

```diff
@@ -108,6 +107,11 @@ func (m *DubboCodec) encodeRequestPayload(ctx context.Context, message remote.Me
 		// todo: kitex mapping
 		Group: "",
 	}
+	// if a method name annotation exists, update the method name to the annotation value.
+	if methodName, ext := m.getMethodAnnotation(message).GetMethodName(); ext {
```

Comment:
`ext` is not a common abbreviation so it's better to spell it fully as `exists`.